.env

```diff
@@ -0,0 +1,57 @@
+# This file holds default settings. They should be sufficient to get a development
+# environment up and running. In some cases the settings will be valid values and suitable
+# for production, such as maximum upload size. In other cases, some settings will be nonsense
+# values, such as access keys.
+#
+# To override these defaults in development (e.g. for developing against an API with real credentials)
+# use the .env.development file, which is git-ignored. To override these settings in production,
+# use the .env.production file, also git-ignored.
```

Comment:
Ah, I love it -- https://github.com/bkeepers/dotenv#what-other-env-files-can-i-use

(The dotenv library for Node very annoyingly enforces [12-factor app architecture](https://www.npmjs.com/package/dotenv#should-i-have-multiple-env-files) which disallows this sort of inheritance)